packages/openapi-generator/src/parser/unique-naming.spec.ts

```diff
@@ -1,69 +1,31 @@
 import { pascalCase } from '@sap-cloud-sdk/util';
 import { ensureUniqueNames } from './unique-naming';
 
-it('ensureUniqueNames replaces duplicate names using defaults (getName, setName, formatName)', () => {
-  const uniqueItems = ensureUniqueNames([
-    { name: 'someDuplicateName1', id: '1' },
-    { name: 'someDuplicateName', id: '2' },
-    { name: 'someDuplicateName', id: '3' }
-  ]);
-
-  expect(uniqueItems).toEqual([
-    { name: 'someDuplicateName1', id: '1' },
-    { name: 'someDuplicateName', id: '2' },
-    { name: 'someDuplicateName2', id: '3' }
-  ]);
+it('ensureUniqueNames replaces duplicate names using defaults', () => {
+  expect(
+    ensureUniqueNames([
+      'someDuplicateName1',
+      'someDuplicateName',
+      'someDuplicateName'
+    ])
+  ).toEqual(['someDuplicateName1', 'someDuplicateName', 'someDuplicateName2']);
```

Comment:
This test is special in a sense that it also checks that the obvious replacement `someDuplicateName1` is already in so it uses a 2. Perhaps you can add more simple test to cover the `[a,a,b]` case and adjust the text here.

Reply:
Ok.